### PR TITLE
refactor: use license components from ui-particules

### PR DESCRIPTION
**Pull request by gcusnieux**
https://gravitee.atlassian.net/browse/CJ-659

### CODE REVIEW

File: gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts [hunk 1]
```diff
@@ -13,186 +13,154 @@
...
-    utm: 'provider_france_connect',
-    image: 'assets/gio-ee-unlock-dialog/am-idp-france-connect.svg',
+  [AmFeature.AM_IDP_FRANCE_CONNECT]: {
+    image: 'assets/gio-license/am-idp-france-connect.svg',
     description:
       'The France Connect identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
-  'am-idp-cas': {
+  [AmFeature.AM_IDP_CAS]: {
     // utm_source=oss_am&utm_medium=provider_cas&utm_campaign=oss_am_to_ee_am
```
Comment (mariano-fernandez) [inline gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts:118]: Not that all `utm`s are gone here, maybe this comment can be also removed

File: gravitee-am-ui/src/styles.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* You can add global styles to this file, and also import other style files */
 @use 'sass:map';
 @use '@angular/material' as mat;
-@use '@gravitee/ui-particles-angular' as gio;
+@use '../node_modules/@gravitee/ui-particles-angular' as gio;
```
Comment (leleueri) [inline gravitee-am-ui/src/styles.scss:4]: is it normal to have this change ?

File: gravitee-am-ui/package.json [hunk 3]
```diff
@@ -37,7 +37,9 @@
     "@angular/platform-server": "^13.3.0",
     "@angular/router": "^13.3.0",
     "@gravitee/ui-components": "^3.41.0",
-    "@gravitee/ui-particles-angular": "^4.11.1",
+    "@gravitee/ui-particles-angular": "7.28.9-cj-659-move-license-5b4624d",
```
[the remark this answers is not in the document]
Comment (gcusnieux) [reply gravitee-am-ui/package.json:40]: Yes. I will update the version just after ui-particules merge/release

File: gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts [hunk 4]
```diff
@@ -13,186 +13,154 @@
...
-    image: 'assets/gio-ee-unlock-dialog/gravitee-ee-upgrade.svg',
+  [AmFeature.AM_IDP_HTTP_FLOW]: {
+    image: 'assets/gio-license/gravitee-ee-upgrade.svg',
     description:
       'The HTTP Flow identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
   },
   // Resources
-  'am-smtp': {
+  [AmFeature.AM_SMTP]: {
     // todo: This resource is missing
```
[the remark this answers is not in the document]
Comment (gcusnieux) [reply gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts]: ✅

File: gravitee-am-ui/src/styles.scss (hunk 2 above)
Comment (gcusnieux) [reply gravitee-am-ui/src/styles.scss]: It's fixed with new configuration https://github.com/gravitee-io/gravitee-access-management/pull/2832/commits/e0dd7b4f2d17103d1e8c5ccc1a8a413b11d4e173#diff-8ba6ed89361dfb033be734a07d69340f45aa031f255cc8de7461608262bb4690R68-R72

File: gravitee-am-ui/src/app/components/navbar/navbar.component.scss [hunk 5]
```diff
@@ -1,5 +1,5 @@
 @use 'sass:map';
-@use '@gravitee/ui-particles-angular' as gio;
+@use 'node_modules/@gravitee/ui-particles-angular' as gio;
```
[the remark this answers is not in the document]
Comment (gcusnieux) [reply gravitee-am-ui/src/app/components/navbar/navbar.component.scss]: It's fixed with new configuration https://github.com/gravitee-io/gravitee-access-management/pull/2832/commits/e0dd7b4f2d17103d1e8c5ccc1a8a413b11d4e173#diff-8ba6ed89361dfb033be734a07d69340f45aa031f255cc8de7461608262bb4690R68-R72

File: gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts (hunk 1 above)
Comment (gcusnieux) [reply gravitee-am-ui/src/app/components/gio-license/gio-license-data.ts]: ✅

**Review by leleueri — APPROVED** [review]

**Review by mariano-fernandez — APPROVED** [review]

**Pull request merged**